src/Microsoft.Framework.Runtime.Hosting/FileFormatException.cs

```diff
@@ -21,6 +21,11 @@ public sealed class FileFormatException : Exception
         public string Path { get; private set; }
         public int Line { get; private set; }
         public int Column { get; private set; }
+        
+        public override string ToString()
+        {            
+            return string.Format("File format error at file {0}, line {1}, column {2}:\n{3}", Path, Line, Column, base.ToString());
```

Comment:
in file?
use string interpolation
consider putting the file name/path in single quotes.